frontend/express/public/stylesheets/vue/clyvue.scss

```diff
@@ -3324,3 +3324,9 @@
         margin: 16px;
     }
 }
+
```

Comment:
can you please move this to /stylesheets/styles/blocks/helpers.scss and @use it in manifest.scss :)
you might have to create the helpers.scss file in blocks.
clyvue.scss will be deprecated once we move to the new ui entirely.

Reply:
done 👍